test-case/composer.json

```diff
@@ -11,6 +11,7 @@
     ],
     "require": {
         "php": "^7.4 || ^8.0",
+        "contao/core-bundle": "self.version",
```

Comment:
You also have to remove it from `require-dev`.

Reply:
done in 4c8fbd0b74e4c84ab2441925f5be4f8b705a0740